### PR TITLE
added persistence data funcionality

### PR DESCRIPTION
**Pull request by soniaigartua**
added persistence data funcionality ussing room library.

Evidence:
Persistence

### CODE REVIEW

File: app/src/main/java/com/example/pps_tudai/activity/MainActivity.java [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 import android.os.Bundle;
 import com.example.pps_tudai.R;
+import com.example.pps_tudai.data.entities.AppRepository;
```
Comment (SartiMau) [inline app/src/main/java/com/example/pps_tudai/activity/MainActivity.java]: do you use these imports here? if not, please delete it

File: app/src/main/java/com/example/pps_tudai/mvp/view/RegistrationView.java [hunk 2]
```diff
@@ -25,6 +22,8 @@
     EditText password;
     @BindView(R.id.et_password2)
     EditText password_repeat;
+//    @BindView(R.id.tv_user)
```
Comment (SartiMau) [inline app/src/main/java/com/example/pps_tudai/mvp/view/RegistrationView.java]: remove commented code pls

File: app/src/main/java/com/example/pps_tudai/mvp/view/RegistrationView.java [hunk 3]
```diff
@@ -25,6 +22,8 @@
...
+//    EditText user_data;
```
Comment (SartiMau) [inline app/src/main/java/com/example/pps_tudai/mvp/view/RegistrationView.java]: same

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -16,4 +16,8 @@
     <string name="register_empty_fields_message">Can not have empty fields</string>
     <string name="register_error_email_message">Introduce a valid email</string>
     <string name="register_error_password_message">The repeated password does not match</string>
+    <string name="no_registration_message">The email address does not exist. Do your registration</string>
+
```
Comment (SartiMau) [inline app/src/main/res/values/strings.xml]: remove unnecessary spaces

Comment (soniaigartua) [reply app/src/main/res/values/strings.xml]: DONE!!!

File: app/src/main/java/com/example/pps_tudai/mvp/view/RegistrationView.java (hunk 3 above)
Comment (soniaigartua) [reply app/src/main/java/com/example/pps_tudai/mvp/view/RegistrationView.java]: DONE!

File: app/src/main/java/com/example/pps_tudai/mvp/view/RegistrationView.java (hunk 2 above)
Comment (soniaigartua) [reply app/src/main/java/com/example/pps_tudai/mvp/view/RegistrationView.java]: DONE!

File: app/src/main/java/com/example/pps_tudai/activity/MainActivity.java (hunk 1 above)
Comment (soniaigartua) [reply app/src/main/java/com/example/pps_tudai/activity/MainActivity.java]: DONE!!

**Pull request merged**